extensions/blocks/social-previews/modal.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Modal styles for Social Previews
+ */
+
+@import '~@automattic/color-studio/dist/color-variables';
+@import '../../../scss/_utilities/mixins/_breakpoint';
+@import '../../../scss/atoms/colors/_colors';
```

Comment:
That's not an option I'm afraid, since those files are not available on WordPress.com.

Reply:
Todo: rewrite using `@import '…extensions/shared/styles/gutenberg-base-styles.scss';`. It provides breakpoint definitions from https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_breakpoints.scss